process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -164,11 +164,11 @@ func (st *storageBootstrapper) loadBlocks() error {
 
 	st.bootstrapper.applyNumPendingMiniBlocks(headerInfo.PendingMiniBlocks)
 
-	processedMiniBlocks := processedMb.NewProcessedMiniBlocks()
-	processedMiniBlocks.ConvertSliceToProcessedMiniBlocksMap(headerInfo.ProcessedMiniBlocks)
-	processedMiniBlocks.DisplayProcessedMiniBlocks()
+	processedMiniBlocksTracker, _ := processedMb.NewProcessedMiniBlocksTracker()
+	processedMiniBlocksTracker.ConvertSliceToProcessedMiniBlocksMap(headerInfo.ProcessedMiniBlocks)
+	processedMiniBlocksTracker.DisplayProcessedMiniBlocks()
 
-	st.blkExecutor.ApplyProcessedMiniBlocks(processedMiniBlocks)
+	st.blkExecutor.SetProcessedMiniBlocksTracker(processedMiniBlocksTracker)
```

Comment:
this is the only place were this set is used. We might convert the Set function into a Get function (only in shard/meta block processor) and call the `ConvertSliceToProcessedMiniBlocksMap` and `DisplayProcessedMiniBlocks` on what the Get function returns. We can easily drop all set functions on the tx coordinator and inner processors.